impl.go

```diff
@@ -447,7 +459,19 @@ to prevent shell globbing.
 		}
 	}
 
-	fns, err := funcs(iface, *flagSrcDir, EmitComments(*flagComments))
+	if *flagRecvPkg == "" {
+		//  "   s *Struct   " , receiver: Struct
+		recvs := strings.Split(strings.TrimSpace(recv), " ") // [0]:s [1]:*Struct
+		receiver := strings.TrimPrefix(recvs[len(recvs)-1], "*")
+		pkg, _, err := typeSpec("", receiver, *flagSrcDir)
+		if err == nil {
+			*flagRecvPkg = pkg.Package.Name
```

Comment:
I'd rather not overwrite the global flag var. Can we use a local var instead?

Reply:
done.